go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/drone-plugins/drone-gcs
 
-go 1.22
+go 1.224
```

Comment:
1.22.4, always run `go mod tidy` after updating

Reply:
update to 1.22.4